python/py-chardet/Portfile

```diff
@@ -18,7 +18,7 @@ long_description    Character encoding auto-detection in Python. \
 checksums           rmd160  dd1063a2449fdd6d775adee29b54eb52194c276b \
                     sha256  ad813ca7b31b19e12c506869ceec53dbd2b9ed5fa73955895394e0245ffe70a2
 
-python.versions     26 27 33 34 35 36 37
+python.versions     26 27 34 35 36 37
```

Comment:
Is leaving 26 here intentional?

Reply:
yes it is for now: the ```canto``` port depends on it. It seems though that this would benefit from an update (see: https://trac.macports.org/ticket/49776), but I haven't looked into this.